api/controllers/issues.js

```diff
@@ -0,0 +1,170 @@
+const mongoose = require('mongoose');
+const path = require('path');
+
+const Issue = require('../models/issue');
+
+exports.issues_get_all = (req, res, next) => {
```

Comment:
You should use _camel case_ when writing javascript. In camel case this would look like `issuesGetAll`.